cmdx.py

```diff
@@ -2627,7 +2674,20 @@ def __str__(self):
         return str(self.read())
 
     def __repr__(self):
-        return str(self.read())
+        cls_name = '{}.{}'.format(__name__, self.__class__.__name__)
+        if self._mplug.attribute().apiType() == om.MFn.kCompoundAttribute:
+            return '{}("{}", "{}")'.format(cls_name,
+                                           self.node().name(),
+                                           self.name())
+        read_val = self.read()
+        if isinstance(read_val, string_types):
+            # Add surrounding single quotes, indicating the value is a string
+            read_val = '"{}"'.format(read_val)
+
+        return '{}("{}", "{}") == {}'.format(cls_name,
+                                             self.node().name(),
+                                             self.name(),
+                                             read_val)
```

Comment:
Alright, added the check here to not include the `read()`ing of the values in case the attribute is compound.

The previous example now looks like the following:
```python
>>> _ = cmds.file(new=True, f=True)
>>> sphere_a = cmds.polySphere(ch=False)[0]
>>> sphere_b = cmds.polySphere(ch=False)[0]
>>> bs = cmds.blendShape(sphere_a, sphere_b)[0]
>>> bs = cmdx.ls(bs)[0]

>>> bs
cmdx.Node("blendShape1")

>>> bs['input']  # A kCompoundAttribute, won't read/show values
cmdx.Plug("blendShape1", "input")

>>> sphere_b = cmdx.ls(sphere_b)[0] 
>>> sphere_b['translate']
cmdx.Plug("pSphere2", "translate") == (0.0, 0.0, 0.0)
```